pages/groups/index.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="vx-row w-full ml-8" id="newsletter-container">
+    <!-- TITLE AND CREATE CARDS? -->
+    <span class="h-15 w-full mr-8 mb-4">
+      <div class = 'inline-block text-black font-bold mb-4 text-ginger-b text-4xl p-4'>
+        YOUR GROUPS
+      </div>
+      <vs-button circle flat success animation-type="vertical" class=" inline-block float-right mr-10 w-20">
+        <div class="text-lg text-xs font-ginger-b">Join</div>
+        <template #animate>
+          <i class="bx bx-plus text-2xl"></i>
+        </template>
+      </vs-button>
+      <vs-button circle flat success animation-type="vertical" class="float-right mr-10 w-20">
+        <div class="text-lg text-xs font-ginger-b">Create</div>
+        <template #animate>
+          <i class="bx bx-plus text-2xl"></i>
+        </template>
+      </vs-button>
+    </span>
+
+    <!-- CARDS -->
+    <div v-if="loaded" class="flex flex-row bg-gray-200" id="groupCard">
+      <GroupCard v-for="(item, index) in groups" :key="index" :group="item"></GroupCard>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'nuxt-property-decorator'
+import { groupsStore } from '~/store'
+import NewsletterModal from '~/screens/newsletter/NewsletterModal.vue'
+import NewsletterCard from '~/components/NewsletterCard.vue'
+import GroupCard from '~/components/GroupCard.vue'
+
+@Component<GroupsPage>({
+  components: { GroupCard },
+  mounted() {
+    console.log('abanana')
```

Comment:
remove

Reply:
done

Reply:
wait no it breaks the things if remove it